src/repos/FlyerRepo.ts

```diff
@@ -62,6 +62,20 @@ const getFlyersBeforeDate = (before: string, limit = DEFAULT_LIMIT): Promise<Fly
   );
 };
 
+const getFlyersByCategorySlug = async (
+  categorySlug: string,
+  limit = DEFAULT_LIMIT,
+  offset = DEFAULT_OFFSET,
+): Promise<Flyer[]> => {
+  return FlyerModel.find({ categorySlug })
```

Comment:
I know this is syntactic sugar but let's make it more explicit, so that we'll be able to update the query appropriately if we ever change the field name to something else (I'm not sure whether the current code will throw a compile time error if the fields are updated): 

```suggestion
  return FlyerModel.find({ categorySlug: categorySlug })
```

Reply:
ESLint wants the shorthand notation...